From 5be0a0c2b42b89f6ad27f85807ddd50adb569711 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Wed, 15 Feb 2023 09:46:35 +0000
Subject: [PATCH 0544/1085] dt-bindings: net: cdns,macb: AXI tuning properties

Add optional properties to tune the AXI interface -
cdns,aw2w-max-pipe, cdns,ar2r-max-pipe and cdns,use-aw2b-fill.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 .../devicetree/bindings/net/cdns,macb.yaml       | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

--- a/Documentation/devicetree/bindings/net/cdns,macb.yaml
+++ b/Documentation/devicetree/bindings/net/cdns,macb.yaml
@@ -131,6 +131,22 @@ properties:
       Node containing PHY children. If this node is not present, then PHYs will
       be direct children.
 
+  cdns,aw2w-max-pipe:
+    $ref: /schemas/types.yaml#/definitions/uint32
+    description:
+      Maximum number of outstanding AXI write requests
+
+  cdns,ar2r-max-pipe:
+    $ref: /schemas/types.yaml#/definitions/uint32
+    description:
+      Maximum number of outstanding AXI read requests
+
+  cdns,use-aw2b-fill:
+    type: boolean
+    description:
+      If set, the maximum number of outstanding write transactions operates
+      between the AW to B AXI channel, instead of the AW to W AXI channel.
+
 patternProperties:
   "^ethernet-phy@[0-9a-f]$":
     type: object
